hack/kind/kind.sh

```diff
@@ -99,6 +99,39 @@ cacert() {
     CAROOT=$(pwd)/.certs mkcert -install
 }
 
+usecockroach() {
+
```

Comment:
How about a big warning message here to say that this isn't expected to work fully at this point? I just want to avoid anyone trying it and then wondering why their env isn't working.